rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,153 @@
+### DCP PR:
+***Leave this blank until the RFC is approved then the Author(s) must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc\#\](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)`
+
+# Technical Support SOP
+
+## Summary
+
+This process RFC describes how the DCP handles technical support requests through the use of the Zendesk application and other processes.
```

Comment:
Why technical support requests, surely we will get scientific requests too and having two different policies feels cumbersome?

Reply:
I agree with @lauraclarke . Some of the questions we've received so far have been scientific questions. For example:

> Can I request for more information on Melanoma infiltration data set that is made public. I request additional information on the causative agent/drug used to induce melanoma and age of the time point specific mouse might also be helpful. 

> I was wondering whether the immune census has associated clustering/celltyping data, that isn't present in the https://preview.data.humancellatlas.org/ page. e.g. a vector containing the cell types (imputed from the gene expression patterns) for each cell ID?

Maybe being clear here that this RFC proposal will handle scientific questions, as well, would be helpful.

Reply:
Hmm, I can't remember if this is part of the SOP, but I think that we need to be clear about types of 'scientific questions' we will answer and what types we won't. I don't think that we should be answering questions that stray into scientific advice (e.g. How should I analyze the data I've downloaded? What clustering method is best? etc). And for the questions @malloryfreeberg listed, I think we should just be pointing these people to the submitters rather than attempting to answer these ourselves.

Reply:
@matthewspeir you are correct that we won't be able to answer all scientific questions and may frequently need to direct people to the contributing lab or a domain expert but that is different from saying we will answer no scientific questions. We should be able to give people to approach with basic questions about the HCA and the data in the DCP and then point them to better resources for more detailed information and we should be able to answer questions about the data we provide and how it was analysed/annotated

Reply:
I agree - I think this proposal was meant to cover the entirety of questions that come in through our data-help@humancellatlas.org email address (the address that feeds into the Zendesk ticketing system). Those can be technical or scientific. Maybe we should exchange the word "technical" here for "user", so "technical support" is just "user support"?

Reply:
+1 for "user support".

As stated, while we may not be able to give answers to all scientific questions, we should have some kind of plan for responses to those types of questions. We don't need to figure out the response now (could be pointing to another resource or the data contributors), but we need to be clear that we will handle these requests in some form.